ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -102,7 +102,7 @@ cd easy-rsa-master/easyrsa3
 
 # Sadly, openssl is very verbose to std*err* with no option to turn it off.
 if ! (./easyrsa --batch init-pki
-      ./easyrsa --batch "--req-cn=${cert_ip}@$(date +%s)" build-ca nopass
+      ./easyrsa --batch "--req-cn=$(echo $(echo ${cert_ip} | cut -b 1-$(expr 64 - $(echo @$(date +%s) | wc -c)))@$(date +%s))" build-ca nopass
```

Comment:
Hmmm, is that need somthing like
```
#      Since the length of CN is limited to 64 bytes, here we cut too long ${cert_ip}
      ./easyrsa --batch "--req-cn=$(echo $(echo ${cert_ip} | cut -b 1-$(expr 64 - $(echo @$(date +%s) | wc -c)))@$(date +%s))" build-ca nopass
#                                  |      |                            |           |<        bytes in tail  >|||            |           
#                                  |      |                            |<  max allowed bytes minus ^        >||             |           
#                                  |      |<   cut ${cert_ip}, if larger than ^                             >|              |           
#                                  |<              Redundant echo from test environment XD                                 >| 
```
or single line enough?